app/templates/views/announcements.cy.html

```diff
@@ -0,0 +1,129 @@
+{% extends "content.html" %}
+
+{%- from "govuk_frontend_jinja/components/breadcrumbs/macro.html" import govukBreadcrumbs -%}
+{%- from "govuk_frontend_jinja_overrides/templates/components/language-select/macro.html" import hmrcLanguageSelect -%}
+{%- from "components/related_content.html" import related_content -%}
+{%- from "components/meta_tags.html" import metaTags -%}
+
+{% set pageTitle = "Profion ar y gweill" %}
```

Comment:
This looks to be the title from the planned tests page, it'll need translating properly, along with anything else new on this page.

Reply:
Ah, good shout. Okay, let's leave this be for now if there are no other issues, and I'll see if I can get those back shortly and added to the PR.